### PR TITLE
Checkout: Order Pay to Template

### PR DESCRIPTION
**Pull request by terence1990**
Put the Order Pay content into a template for easy over-riding. This will have no affect to existing users as it there is a template fallback.

### CODE REVIEW

File: templates/checkout/order-pay.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	</li>
+	<li class="date">
+		<?php _e( 'Date:', 'woocommerce' ); ?>
+		<strong><?php echo date_i18n(get_option('date_format'), strtotime($order->order_date)); ?></strong>
+	</li>
+	<li class="total">
+		<?php _e( 'Total:', 'woocommerce' ); ?>
+		<strong><?php echo $order->get_formatted_order_total(); ?></strong>
+	</li>
+	<?php if ($order->payment_method_title) : ?>
```
Comment (WPprodigy) [inline templates/checkout/order-pay.php]: Should use `$order->get_payment_method_title()` I think. https://github.com/woocommerce/woocommerce/blob/c16acc6b5104acb0ed082e7df1c63dfd77598459/includes/class-wc-order.php#L668

File: templates/checkout/order-pay.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	<li class="method">
+		<?php _e( 'Payment Method:', 'woocommerce' ); ?>
+		<strong><?php
+			echo $order->payment_method_title;
```
Comment (WPprodigy) [inline templates/checkout/order-pay.php]: Would use `$order->get_payment_method_title()` here as well then.

File: includes/shortcodes/class-wc-shortcode-checkout.php [hunk 3]
```diff
@@ -130,35 +130,10 @@ private static function order_pay( $order_id ) {
...
-						<?php endif; ?>
-					</ul>
-
-					<?php do_action( 'woocommerce_receipt_' . $order->get_payment_method(), $order_id ); ?>
-
-					<div class="clear"></div>
-					<?php
-
+					wc_get_template( 'checkout/order-pay.php', array(
+						'order'              => $order
```
Comment (WPprodigy) [inline includes/shortcodes/class-wc-shortcode-checkout.php]: No need for all the space between the key/value in the array :)

File: templates/checkout/order-pay.php (hunk 2 above)
Comment (claudiosanches) [inline templates/checkout/order-pay.php]: And should keep `wp_kses_post()`.

**Review by claudiosanches — CHANGES_REQUESTED** [review]
Creating a template for this is just fine, but not sure the reason from change the data removing important things and using deprecated arguments from `WC_Order` class... Missing fix coding standards too.
Let me know when you fixed this things.
Thanks for contribute.

**Comment by terence1990** [comment]
Changes have been made according to all of your comments, @claudiosanches i think my original commits were made based on an old master branch, have correct order-pay template to use what's current in the master branch

**Comment by mikejolley** [comment]
@claudiosanches Is the naming of this template accurate? I mean it contains order details but there is nothing about payment in there?

**Comment by terence1990** [comment]
@mikejolley hmm yeah it's not contextual enough, the template is for the view usually just before you get redirected to the vendor.

In my use case i am dealing with Sage 3D Secure on this page. Not really sure what to call this view as it is so transitional.

**Comment by terence1990** [comment]
@mikejolley I have changed the name to order-receipt to reflect the actions / hooks naming.

**Pull request merged**